### PR TITLE
TILA-1518: Test fix and pull request template

### PR DESCRIPTION
**Pull request by saaste**
## Change log
- Fixed flaky `test_filter_by_multiple_reservation_unit` test
- Added simple pull request template

## Deployment reminder
- [ ] PR requires deployment changes
- [ ] Deployment PR done [link here]
- [ ] Library updated if needed
- [ ] PR merged

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- Add changes
+- Here
+
+## Other notes
+Additional notes if needed
+
+## Deployment reminder
+- [ ] PR requires deployment changes
+- [ ] Deployment PR done [link here]
+- [ ] Library updated if needed
```
Comment (nicobav) [inline .github/pull_request_template.md:11]: Azure devops library updated if needed 
:monocle_face:

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Change log
+- Add changes
+- Here
+
+## Other notes
+Additional notes if needed
+
+## Deployment reminder
+- [ ] PR requires deployment changes
+- [ ] Deployment PR done [link here]
```
Comment (nicobav) [inline .github/pull_request_template.md:10]: What's the Deployment PR?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- [ ] Library updated if needed
+- [ ] PR merged
```
Comment (nicobav) [inline .github/pull_request_template.md:12]: This PR?

File: .github/pull_request_template.md (hunk 2 above)
Comment (saaste) [reply .github/pull_request_template.md:10]: That is the PR needed to change the configuration in the `tilauspalvelu-pipelines` repository. So basically every time we add new configuration keys. For example [this](https://dev.azure.com/City-of-Helsinki/tilavarauspalvelu/_git/tilavarauspalvelu-pipelines/pullrequest/2113).

File: .github/pull_request_template.md (hunk 3 above)
Comment (saaste) [reply .github/pull_request_template.md:12]: No, pipeline PR.
This requires some re-wording 😄

**Review by nicobav — APPROVED** [review]
Looking goooood!

**Pull request merged**